README.md

```diff
@@ -345,3 +345,14 @@ If you want to contribute to this project, see [Contributing Guide](https://gith
+| `akka.actor.allow-java-serialization=on`  | Serialization succeeds - but using Java serialization. Warning log like below appears on the startup:<br><br> [WARN] [...] Using the default Java serializer for class [sample.cluster.transformation.TransformationResult] which is not recommended because of performance implications. Use another serializer or disable this warning using the setting 'akka.actor.warn-about-java-serializer-usage'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Serialization succeeds - but using Java serialization (which is not a good choice for production). Warning log like below will appear on the startup:<br><br> [WARN] [...] Using the default Java serializer for class [sample.cluster.transformation.TransformationResult] which is not recommended because of performance implications. Use another serializer or disable this warning using the setting 'akka.actor.warn-about-java-serializer-usage'                                                                                                                                                 |
+
+**Note** - the default setting for Akka 2.5 is `akka.actor.allow-java-serialization=on`, whereas `akka.actor.allow-java-serialization=off` is the default setting for Akka 2.6
```

Comment:
Ouch! that's the reason why I had the impression that 2.5 and 2.6 differ in their behavior (although in fact they handle the `on`/`off` cases almost identically... just the different case is the default)